packages/suite/src/views/wallet/trading/common/TradingForm/TradingFormApproval.tsx

```diff
@@ -196,7 +196,7 @@ export const TradingFormApproval = ({
             payload: {
                 type: 'exchange-form',
                 action: 'approve',
-                ...cryptoInfo,
+                ...getCryptoInfo(),
```

Comment:
Same here

Reply:
No, this is intentional and better. https://github.com/trezor/trezor-suite/pull/23314#discussion_r2580808530